File: src/main/java/me/miki/shindo/management/mods/impl/ShinyPotsMod.java
package me.miki.shindo.management.mods.impl;

import me.miki.shindo.management.language.TranslateText;
import me.miki.shindo.management.mods.Mod;
import me.miki.shindo.management.mods.ModCategory;

public class ShinyPotsMod extends Mod {

    private static ShinyPotsMod instance;

    public ShinyPotsMod() {
        super(TranslateText.SHINY_POTS, TranslateText.SHINY_POTS_DESCRIPTION, ModCategory.RENDER);

        instance = this;
    }

    public static ShinyPotsMod getInstance() {
        return instance;
    }
}

File: src/main/kotlin/me/miki/shindo/management/mods/impl/ShinyPotsMod.kt
package me.miki.shindo.management.mods.impl

import me.miki.shindo.management.language.TranslateText
import me.miki.shindo.management.mods.Mod
import me.miki.shindo.management.mods.ModCategory
import me.miki.shindo.management.nanovg.font.Lucide
import me.miki.shindo.management.nanovg.font.Shinconic

class ShinyPotsMod : Mod(TranslateText.SHINY_POTS, TranslateText.SHINY_POTS_DESCRIPTION, ModCategory.RENDER, Shinconic.MOD_SHINY_POTS) {
    init {
        instance = this
    }

    companion object {
        @JvmField
        var instance: ShinyPotsMod? = null
    }
}
